terraform-aws-dataworks-pdm.tf

```
@@ -0,0 +1,69 @@
+resource "github_repository" "aws-dataworks-pdm" {
+  name             = "aws-dataworks-pdm"
```

Doesn't need AWS in front of the name.

```suggestion
  name             = "dataworks-pdm"
```